_topic_maps/_topic_map.yml

```diff
@@ -3576,19 +3576,24 @@ Topics:
   Topics:
   - Name: Distributed tracing architecture
     File: distr-tracing-architecture
-- Name: Distributed tracing installation
-  Dir: distr_tracing_install
-  Topics:
-  - Name: Installing distributed tracing
-    File: distr-tracing-installing
-  - Name: Configuring the distributed tracing platform
-    File: distr-tracing-deploying-jaeger
-  - Name: Configuring distributed tracing data collection
-    File: distr-tracing-deploying-otel
-  - Name: Upgrading distributed tracing
-    File: distr-tracing-updating
-  - Name: Removing distributed tracing
-    File: distr-tracing-removing
+- Name: Distributed tracing platform - Jaeger
```

Comment:
Should we use camel case for the topic names ? Like Distributed Tracing Platform - Jaeger and Distributed Tracing Data Collection - OpenTelemetry.

Reply:
The official name of the product is `Red Hat OpenShift distributed tracing platform` (lower case after `OpenShift`), but here I guess we can use capitalization as we want. 

I will leave this to docs team to decide